timefold-solver-python-core/src/main/python/config/_config.py

```diff
@@ -378,6 +378,29 @@ def _to_java_solver_config_override(self):
         return out
 
 
+@dataclass(kw_only=True)
+class SolverManagerConfig:
```

Comment:
Why aren't we adding the `threadFactoryClass` attribute?

Would this affect the translation between Java and Python?